### DISCUSSION TITLE
docker配置SSL证书

### DISCUSSION THREAD

**Discussion by Joyrocky** [natfrp/wiki#315] [使用问题]
使用`docker`配置`SSL`证书，阿里云的`Nginx`的`SSL`证书为`pem/key`文件，与文档有一些出入；还有`docker`的部署的根目录与容器的`Dockerfile`有些不同，导致挂载`SSL`证书不知道是否出错；还有请问如何进入`docker`容器呢，我想使用`docker exec -it sakura1 bash`和`docker exec -it sakura1 sh`都无法进入docker内部查看目录结构😂
```bash
docker run -d --restart=always --pull=always --name=sakura1 --mount type=bind,source=/home/docker/sslcert/aliyun_rabbit.crt,target=/run/frpc/aliyun_rabbit.crt --mount type=bind,source=/home/docker/sslcert/aliyun_rabbit.key,target=/run/frpc/aliyun_rabbit.key natfrp/frpc -f XXXXXX --remote_control YYYYY
```
![image](https://user-images.githubusercontent.com/23481533/233888658-50ece11f-c8f3-484c-8215-04b00928791a.png)
![image](https://user-images.githubusercontent.com/23481533/233890081-ff3ccc76-e85f-42c0-bda4-af5a5bbeb108.png)

**Comment by fakeboboliu** [comment]
很抱歉因为一些推送问题，我们刚刚看到这个问题，

请确保从正确的源拉取镜像，即我们在文档中提供的源，
使用非 docker hub/ghcr/指定镜像url 拉取的镜像可能存在问题，
尤其是使用镜像加速器时。
